migrations/Version202407021254084106_pciSamples.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+  declare(strict_types=1);
+
+  namespace oat\pciSamples\migrations;
+
+  use Doctrine\DBAL\Schema\Schema;
+  use oat\tao\scripts\tools\migrations\AbstractMigration;
```

Comment:
```suggestion
  use Doctrine\DBAL\Schema\Schema;
  use Doctrine\Migrations\Exception\IrreversibleMigration;
  use oat\pciSamples\scripts\install\RegisterPciTextReaderIMS;
  use oat\tao\scripts\tools\migrations\AbstractMigration;
```
You need to `use` all classes you reference in the script, or the migration will error.